### ISSUE TITLE
How to use resnet50 model in tf_cnn_benchmarks.py?

### ISSUE THREAD

**Issue by ljppro** [NervanaSystems/ngraph-tf#89]
Hi,
I download the savedmodel and checkpoints from tensorflow/models and untar them into a single directory. But when I run this benchmark, an error reported as:
NotFoundError (see above for traceback): Key v0/cg/affine0/biases not found in c                                                                             heckpoint
         [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_INT64, DT_FLOAT, DT_FLOAT                                                                             , DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _d                                                                             evice="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/                                                                             RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

I tried another models from other sites also. But the almost same error is there.

What resnet model can work with this benchmark and how should I specify the data_dir parameter?

Thanks!

**Comment by sharathns93** [comment]
Hi,
Thanks for trying out the ngraph-tf bridge. The pretrained models downloaded from tensorflow/models repo are  unfortunately not compatible with tf_cnn_benchmarks.py script. 

However you can train a model locally with dummy data for a few iterations using this command:
  python tf_cnn_benchmarks.py --batch_size 16 --model resnet50 --train_dir <path_to_resnet_model> --num_batches=10 --data_format NHWC --select_device CPU

and then use the pretrained model generated here for evaluation. 
Here is the command line: 
python tf_cnn_benchmarks.py --batch_size 128 --model resnet50 --train_dir <path_to_resnet_model> --num_batches=1000   --data_format NCHW --eval --select_device NGRAPH

For best performance on  2-socket Intel(R) Xeon(R) Platinum 8180 add the following before running the command:
KMP_BLOCKTIME=20 OMP_NUM_THREADS=28  KMP_AFFINITY=granularity=fine,proclist=[0-27]

Please note that since the model is trained with dummy data, the accuracy numbers generated are random and not correct. You can also train it with real data (tf_records for Imagenet or CIFAR10) to completion and then verify to get the right accuracy numbers.
